livekit-plugins/livekit-plugins-assemblyai/livekit/plugins/assemblyai/stt.py

```diff
@@ -116,13 +117,45 @@ def stream(
         conn_options: APIConnectOptions = DEFAULT_API_CONNECT_OPTIONS,
     ) -> "SpeechStream":
         config = dataclasses.replace(self._opts)
-        return SpeechStream(
+        self._active_speech_stream = SpeechStream(
```

Comment:
This isn't right since you could have multiple streams running. 
Here is how I did it for VAD: https://github.com/livekit/agents/blob/57834ec97db4a23452e52f98b2b48fac37eed0aa/livekit-plugins/livekit-plugins-silero/livekit/plugins/silero/vad.py#L137